File: src/sharing/service/disLike/impl/DisLikeService.java
package sharing.service.disLike.impl;

import sharing.entity.disLike.DisLike;

public interface DisLikeService {

	public Long addDisLike(DisLike disLike) throws Exception;
	
	public DisLike updateDisLike(DisLike disLike) throws Exception;
	
	public DisLike findDisLikeById(Long disLikeId) throws  Exception;
	
	public Long deleteDisLikeById(Long disLikeId) throws Exception;
	
	public String findDisLikeByUserIdAndDynamicId(Long userId, Long dynamicId) throws Exception;
}

File: WebContent/app/service/disLike/DisLikeService.js
define(['app'],function(app) {
	app.factory('DisLikeService',function($resource) {
		var disLikeService = $resource('../DisLikeAction/:action', {});
		
		disLikeService.addDisLike = function(disLike, sucesscb, errorcb) {
			disLikeService.save({action:"addDisLike"},{disLike:disLike}, sucesscb, errorcb);
		},
		
		disLikeService.findDisLikeById = function(disLikeId,sucesscb,errorcb) {
			disLikeService.get({action:"findDisLikeById",disLikeId:disLikeId},sucesscb,errorcb);
		},
		
		disLikeService.updateDisLike = function(disLike,sucesscb,errorcb) {
			disLikeService.save({action:"updateDisLike"},{disLike:disLike},sucesscb,errorcb);
		},
		
		disLikeService.deleteDisLikeById = function(disLikeId,sucesscb,errorcb)	{
			disLikeService.save({action:"deleteDisLikeById"},{disLikeId:disLikeId},sucesscb,errorcb);
		},
		disLikeService.findDisLikeByUserIdAndDynamicId = function(userId, dynamicId, sucesscb, errorcb) {
			disLikeService.get({action:"findDisLikeByUserIdAndDynamicId",userId:userId, dynamicId:dynamicId},sucesscb,errorcb);
		}
		return likeService;
	})
})